### DISCUSSION TITLE
Exporting multiple notebooks to html-wasm

### DISCUSSION THREAD

**Discussion by aszenz** [marimo-team/marimo#3178] [Q&A] [answered]
Exporting a single html-wasm notebook works great, how would one export multiple notebooks and deploy them on a single static site. 

Can different notebooks share the assets folder?

Will something like this work after renaming index.html of each notebook: 

```
assets
nb1.html
nb2.html
```

So 
mysite.com/nb1.html and 
mysite.com/nb2.html works

Also how can it integrate with marimos built in file viewer to show other notebooks on the site?

**Comment by aszenz** [comment, marked as answer]
Looks like this works already, assets can be shared between notebook html files, pyodide and python packages are loaded on the fly from cdns.

**Reply by mscolnick** [reply]
yep they can. that is the recommended way to do it (that's what we do: export then rename the index.html)

**Reply by mscolnick** [reply]
You no longer need to rename the index.html, you can just do 
```bash
marimo export html-wasm nb.py -o out/nb.html
```
